testsuite/features/step_definitions/api_common.rb

```diff
@@ -500,16 +496,8 @@ def wait_action_complete(actionid, timeout: DEFAULT_TIMEOUT)
   step %(I should get status "#{status}" for system "#{get_system_id(node)}")
 end
 
-Then(/^I should get the test base channel$/) do
-  arch, _code = get_target('server').run('uname -m')
-  arch.chomp!
-  channel = if arch != 'x86_64'
-              'fake-base-channel-i586'
-            else
-              'fake-base-channel'
-            end
-  log "result: #{@result}"
-  assert(@result['channel_labels'].include?(channel))
+Then(/^I should get the "([^"]*)" channel label$/) do |channel_label|
+  assert(@result['channel_labels'].include?(channel_label))
```

Comment:
Eric will be happy to see the old step definition gone. We recently talked about that in a differnt PR, if you remember.